lambdas/abandon/src/services/abandon-dynamo-service.ts

```diff
@@ -1,29 +1,25 @@
 import { getRecordBySessionId } from "../../../common/src/database/get-record-by-session-id";
 import { SessionItem } from "../../../common/src/database/types/session-item";
 import { DynamoDBClient, UpdateItemCommand } from "@aws-sdk/client-dynamodb";
-import { RecordExpiredError, RecordNotFoundError } from "../../../common/src/database/exceptions/errors";
+import { RecordNotFoundError } from "../../../common/src/database/exceptions/errors";
 import { CriError } from "../../../common/src/errors/cri-error";
 import { logger } from "../../../common/src/util/logger";
 
 const dynamoClient = new DynamoDBClient();
 
 export async function retrieveSessionRecord(sessionTableName: string, sessionId: string) {
   try {
-    const [sessionItem] = await getRecordBySessionId<SessionItem>(
+    const sessionItem = await getRecordBySessionId<SessionItem>(
```

Comment:
Not so sure about having an abandon service, getRecordBySessionId can be used in the lambda directly, generally I love getRecordBySessionId to be used as is not wrappers around it,  also the abandon-audit-service.ts is it really that specific,  I suppose it would nice to make audit stuff more generic, make certain parts optional, something that knows how to send stuff through eventbridge. This is not horrible just my opinion something like https://github.com/govuk-one-login/ipv-cri-common-lambdas/blob/main/lambdas/src/common/services/audit-service.ts

Reply:
Agreed on some of your points, let's defer the discussion until we're closer to establishing a standard for these Lambdas